common_test.go

```diff
@@ -28,3 +33,77 @@ func TestComputePayloadAuthenticator(t *testing.T) {
 		}
 	}
 }
+
+func runTestOverVersions(t *testing.T, f func(t *testing.T, version Version)) {
+	for _, version := range KnownVersions() {
+		version := version // capture range variable.
+		t.Run(version.String(), func(t *testing.T) {
```

Comment:
I think so. It looks like:

```
    --- PASS: TestEncrypt/SealAndOpenSmall (0.00s)
        --- PASS: TestEncrypt/SealAndOpenSmall/1.0 (0.00s)
        --- PASS: TestEncrypt/SealAndOpenSmall/2.0 (0.00s)
```

The only downside is if you want to match subtests in `-run` you have to explicitly put slashes in or something.